index.js

```diff
@@ -205,5 +205,8 @@ module.exports = {
 
     // Airbnb enforces this but we use this feature a lot!
     'react/jsx-props-no-spreading': 'off',
+
+    // Disable camelcase rule. 
+    "@typescript-eslint/camelcase": ["off"]
```

Comment:
we probably want to use single quotes here, and add a trailing comma.

Reply:
Yeah, I see now the tests are failing and it looks like it's upset about the syntax. When I tested this I put the code directly into `code-manager`'s file first, and I think it wanted that syntax, so it was less than ideal when I copied it over into this repo. I'll fix this.